### PR TITLE
Nf 290 cya tidy

### PR DESCRIPTION
**Pull request by njmoorewitchcat**

### CODE REVIEW

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/components/subHeading.scala.html [hunk 1]
```diff
@@ -16,4 +16,4 @@
 
 @(content: String)
 
-<h2 class="govuk-heading-m">@{Text(content).asHtml}</h2>
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/components/subHeading.scala.html:19]: Is this subHeading component used elsewhere (i.e. not on CYA) and if so us that spacing appropriate there as well?

File: conf/messages [hunk 2]
```diff
@@ -254,7 +254,10 @@ check_answers.payment.title = Payment information
 check_answers.payment.payTo = Who is to be repaid?
 check_answers.payment.payTo.Importer = Importer
 check_answers.payment.payTo.Representative_of_importer = Representative
-check_answers.payment.bankDetails = Bank details
+check_answers.payment.bankDetails = Name on the account<br>Sort code<br>Account number
```
Comment (SteveSugden) [inline conf/messages:257]: I questioned this label when I originally did bank details.  IMO having three "labels" like this is not right.  Also having mark-up in the message is not ideal.   I personally would push back on the design but if we have to implement it like this then have three messages and do the formatting in the view

**Review by SteveSugden — APPROVED** [review]
LGTM - just a couple of clarifications/suggestions.

Have you checked the ATs are still ok?  They rely on the pseudo class names for the CYA sections (if you've not changed the names then all should be OK)

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/components/subHeading.scala.html (hunk 1 above)
Comment (njmoorewitchcat) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/components/subHeading.scala.html:19]: SubHeading is used in CreateCheckYourAnswers and AmendCheckYourAnswers only for the headers between sections.  If we decide to add it to other pages, we probably want to create a special CheckYourAnswers subHeading or something else then.  At the moment its safe.

File: conf/messages (hunk 2 above)
Comment (njmoorewitchcat) [reply conf/messages:257]: I will get a steer from design team.  I agree 100% with you on the better 'Bank details'.  And yeah, brain dead when I implemented, your are correct, should be three seperate messages - If I need to change it based on the feedback, I will do that.

**Comment by njmoorewitchcat** [comment]
> LGTM - just a couple of clarifications/suggestions.
> 
> Have you checked the ATs are still ok? They rely on the pseudo class names for the CYA sections (if you've not changed the names then all should be OK)

Will check them now - I have removed a section, so they may not be ok..

File: conf/messages (hunk 2 above)
Comment (njmoorewitchcat) [reply conf/messages:257]: Changing this back - we will have a longer discussion with the design team on deviations from the prototype

**Review by SteveSugden — APPROVED** [review]

**Pull request merged**